src/main/webapp/app/entities/grade-step.model.ts

```diff
@@ -25,4 +25,6 @@ export class GradeStepsDTO {
     public gradeType: GradeType;
     public gradeSteps: GradeStep[];
     public maxPoints?: number;
+    public plagiarismGrade: string;
```

Comment:
Why arent we using enums or TS- template literal types?
`type PlagiarismGrade = 'unterschleif' | 'other_possibility' | 'asd'`

Reply:
Since it is freely editable by the instructor, we need to handle any string to be sure.